src/asttypename.d

```diff
@@ -0,0 +1,112 @@
+/**
+ * Part of the Compiler implementation of the D programming language
+ * Copyright:   Copyright (c) 1999-2017 by The D Language Foundation, All Rights Reserved
+ * Authors:     Stefan Koch
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(DMDSRC _asttypename.d)
+ */
+
+module ddmd.asttypename;
```

Comment:
there are no underscores in the rest of the dmd sources.
execpt for `ti_chars` and `objc_glue`
plus this is not a module which is meant to be commonly imported